examples/runtime-sdk/rofl-oracle/src/main.rs

```diff
@@ -50,17 +53,17 @@ impl OracleApp {
         let observation = tokio::task::spawn_blocking(move || -> Result<_> {
             // Request some data from Coingecko API.
             let rsp: serde_json::Value = rofl_utils::https::agent()
-                .get("https://api.coingecko.com/api/v3/simple/price?ids=oasis-network&vs_currencies=USD")
+                .get("https://www.binance.com/api/v3/ticker/price?symbol=ROSEUSDT")
```

Comment:
Why this change?

Reply:
Coingecko has a very low rate limit for the free/demo plan (30 requests/min) which doesn't work long for the Localnet's 1s block time. Binance does not have such a limit for getting price quotes.

Reply:
We could also add a rate limit in the demo, there is no need to query so often.